### DISCUSSION TITLE
大佬，关于青龙运行，本人在终端进行了dotnet的安装，跑脚本依旧是下面的报错，在青龙环境里也安装了dotnet，同样也是报错，想咨询下是什么原因，感谢大佬，不知道是不是ck的问题

### DISCUSSION THREAD

**Discussion by Doomscrolls** [RayWangQvQ/BiliBiliToolPro#270] [General]
![1661936667471](https://user-images.githubusercontent.com/89169723/187643980-6f0487ed-4298-4203-a0fe-41ab16e5c60e.png)
![1661936642762](https://user-images.githubusercontent.com/89169723/187643988-96ee93a9-9169-43fb-a059-d4ba619067cf.png)

**Comment by SnowIF** [comment]
遇到过同样的问题，不过解决了。按照教程走完之后 打开青龙面板，点依赖管理，然后新建依赖。依赖类型nodejs，名称dotnet。最后重启docker。就可以了，我不知道我说的对不对，但是我是这样解决的。
